packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/options/components/activation-status/activation-status.controller.js

```diff
@@ -11,6 +11,11 @@ export default class {
   }
 
   onStatusChange() {
+    if (this.statusName === 'enabling') {
+      this.statusName = 'beingActivated';
+    } else if (this.statusName === 'disabling') {
+      this.statusName = 'beingDeactivated';
+    }
```

Comment:
**nitpick**:  component properties should not be overridden
I see three solutions :
1. Create aliases in the constant
```javascript
ACTIVATION_STATUS.enabling = ACTIVATION_STATUS.beingActivated;
ACTIVATION_STATUS.disabling = ACTIVATION_STATUS.beingDeactivated;
```
2. Create a computed data
```javascript
get computedStatusName() {
  const { statusName } = this;
  if (statusName === 'enabling') return 'beingActivated';
  if (statusName === 'disabling') return 'beingDeactivated';
  return statusName;
}
```
3. Transform the source objects carrying the status just after the API sent them, but it would involve too many changes and possible side effects

Reply:
Thanks for your suggestions, will go with the first option, seems the cleanest way to do it.

Reply:
Done, has required a small change in the activation-status.html to handle translations correctly